CMakeLists.txt

```diff
@@ -243,6 +244,8 @@ ExternalProject_Add(psi4-core
               -Dgau2grid_DIR=${gau2grid_DIR}
               -Dgdma_DIR=${gdma_DIR}
               -DLibint_DIR=${Libint_DIR}
+              -DLibint2_DIR=${Libint2_DIR}
+              -Dlibint2_DIR=${Libint2_DIR}
```

Comment:
Line 248 repeats 247.

Reply:
sadly, not quite. one's `L` and one's `l` b/c we're dealing with two ppl's cmake at the moment. you can use the EFV's `make export` that's gnumake+cmake or my pure cmake. definitely there will be only one in the end.

Reply:
Oh, right.... I didn't see the difference in capitalization the first go round.